### PR TITLE
issue-146-1 cob docker test support overlay compose files

### LINKED ISSUE #146: cob docker test - support loading additional compose files
Today, cob docker test generates a compose file containing only cob subsystems (worker, db, wsgi, ..)
In my project i would like to also add another container for testing so that i can run in a fully contained env, not letting my tests communicate with the outside world.

### my thoughts are:
**cmdline would look like**: `cob docker test <.. today's args> --overlay-compose-file <additional compose file>`
**under the hood**: the new container/s should be added to the default testing network docker creates (was not aware of that but it seems it happens automatically if using docker-compose with multiple compose files)

### PR DESCRIPTION
**Pull request by sivanbecker**
@vmalloc 
sorry for the mess of pushing a new branch for issue 146 ( preferred making changes over clean slate instead of rebasing)

resolves #146

### CODE REVIEW

File: cob/project.py [hunk 1]
```diff
@@ -43,6 +43,7 @@ def __init__(self, root='.'):
         self.tests = os.path.abspath(root) + '/tests'
         self._static = {}
         self.tst_cfg_dir = self._tst_cfg_dir
+        self.ocf_dir = self._ocf_dir
```
Comment (vmalloc) [inline cob/project.py:46]: What is ocf_dir?

Comment (vmalloc) [reply cob/project.py:46]: This is a very confusing behavior - taking the result of a property and putting it in an attribute with a very similar name...

Comment (sivanbecker) [reply cob/project.py]: this is indeed confusing and not needed. i fixed it.

Comment (sivanbecker) [reply cob/project.py]: In my last commit i also removed the _tst_cfg_dir which had the exact same issue.
in both cases i just left the properties.

**Pull request merged**